src/client/stylesheets/style.less

```diff
@@ -378,3 +378,11 @@ hr.wide {
 		width: 100%;
 	}
 }
+
+
+@media only screen and (max-width: 768px){
+	.BookBrainz > .container-fluid > .navbar-header{
+		margin-left: -13px;
+	}
```

Comment:
This seems to work, but I think it would be  better to target the `navbar-brand` element instead.
The idea is to keep the `navbar-header` as full width, and rely on the `container-fluid`'s padding, the same that sets the space of the hamburger button on the right side.
Here's what your proposal looks like if I visualize the margins/padding:
<img width="738" alt="Capture d’écran 2020-02-16 à 12 44 27" src="https://user-images.githubusercontent.com/6179856/74612700-b0e39500-50bc-11ea-935e-75a914d57324.png">
You can see there's an empty space on the left.

Instead, I propose to set the `navbar-brand` element's margin-left to 0 (currently it is set to -26px), which would look like this:
<img width="736" alt="Capture d’écran 2020-02-16 à 12 45 07" src="https://user-images.githubusercontent.com/6179856/74612738-0ddf4b00-50bd-11ea-983f-7353c4aabca3.png">

The padding for the container-fluid and the navbar-brand would collapse (they count as one)
<img width="735" alt="Capture d’écran 2020-02-16 à 12 44 58" src="https://user-images.githubusercontent.com/6179856/74612737-0cae1e00-50bd-11ea-8a99-36f2708bc32a.png">


The navbar-brand's negative margin is currnetly targetted with these CSS selectors: `body.BookBrainz .navbar .navbar-brand.logo`. Adding `> .container-fluid` in there like you did will works nicely :)

Reply:
hey @monkeyDo !

i are right!!!

but i got  one more implementation for the solution that is more i robust and intuitive as i think.( almost borrowed from your idea);


right now, we have following elements structure 

![container-fluid](https://user-images.githubusercontent.com/43696525/74645893-c92be080-519e-11ea-9a73-4355cb589b3b.png)

here , "container-fluid" maintains 21px distance from left as well as right which is ok to make things symmetric!

so "navbar-header" should not be given any negative margin after that , otherwise our structure won't look symmetric anymore!

so i will remove all negative margin from "navbar-header" , only problem is with "toggle-button"  which will have extra distance from right because of "container-fluid" and its own margin
21px + 21px  = 42px in total!!!  that really sucks!!!! 

![extramarginfortogglebutton](https://user-images.githubusercontent.com/43696525/74647365-8b7c8700-51a1-11ea-974a-43dee62fc78c.png)





:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::now the solution is :::::::::::::::::::::::::::::::::::::::::::::::::::::::

 1. i will remove every negative margin on "navbar-header".
2. i will remove extra 21px margin from "toggle-button" so that it would be only at a distance of 21px from right which is due to "container-fluid" !!
that's it

Reply:
@MonkeyDo 
 i have done the promised changes .
and now it looks very neat and clean with "NO NEGATIVE STUFFS"
and it is more symmetric than before!!!!